marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -80,6 +89,22 @@ handleQuery (ReceiveWebSocketMessage msg next) = do
       (ErrorResponse error) -> pure unit
   pure $ Just next
 
+-- FIXME: We need to discuss if we should remove this after we connect to the PAB. In case we do,
+--        if there is a disconnection we might lose some seconds and timeouts will freeze.
```

Comment:
Hmm, yeah, let's give it some thought. My immediate inclination is to remove this and just have a single source of truth for the slot number (the PAB). If we lose the connection to the backend, we should show a general error/warning somewhere - since that will make trouble quite generally, not only with the slot number.

Reply:
The problem of the source of thruth being the PAB is that I need to pass that value around. In this PR there are some calls to `currentSlot :: Effect Slot` where I have access to doing some effects.

My inclination would be to compare the PAB slot with the one from the system and raise a warning somehow if they differ by more than 5 seconds or something.